# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: Fix yesexpr/noexpr/yesstr/nostr
# DP: Related bugs: #295810
# DP: Dpatch author: Denis Barbier <barbier@debian.org>
# DP: Patch author: eric pareja <xenos at upm edu ph>
# DP: Upstream status: not submitted
# DP: Date: 2006-01-10

---
 localedata/locales/tl_PH |   14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

--- a/localedata/locales/tl_PH
+++ b/localedata/locales/tl_PH
@@ -140,7 +140,19 @@
 END LC_TIME
 
 LC_MESSAGES
-copy "en_US"
+
+% "^[oOyY].*" for "Yes"
+yesexpr "<U005E><U005B><U006F><U004F><U0079><U0059><U005D><U002E><U002A>"
+
+% "^[hHnN].*" for "No"
+noexpr "<U005E><U005B><U0068><U0048><U006E><U004E><U005D><U002E><U002A>"
+
+% "Oo" for "Yes"
+yesstr "<U004F><U006F>"
+
+% "Hindi" for "No"
+nostr "<U0048><U0069><U006E><U0064><U0069>"
+
 END LC_MESSAGES
 
 LC_PAPER
